curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-debugging-by-building-a-random-background-color-changer/66326913b729e16dd0127a36.md

```diff
@@ -0,0 +1,237 @@
+---
+id: 66326913b729e16dd0127a36
+title: Step 7
+challengeType: 0
+dashedName: step-7
+---
+
+# --description--
+
+CamperBot has finished building out their random background color changer. However, when they click the button, the background color does not change. 
+
+It looks like they are trying to use the `onclick` property but they are using it incorrectly.  The `onclick` property should be assigned a function reference.
+
+Fix the error in the `btn.onclick = changeBackgroundColor();` line.
+
+Remember that you worked with the `onclick` property in the Role playing game project. Look back at the final solution to see how `onclick` was properly used.
+
+Once you fix that final bug, the random background color changer will be complete!
+
+# --hints--
+
+You should not call the `changeBackgroundColor` function. You should instead assign the function reference to the `onclick` property.
```

Comment:
This would work better with a better instead of a comma, IMO.

> You should not call the `changeBackgroundColor` function. Instead, assign the function reference to the `onclick` property.